### ISSUE TITLE
1) molecular entity in the WG3 model doesn’t have the formula information or other information that PDB provides

### ISSUE THREAD

**Issue by proccaserra** [biocaddie/WG3-MetadataSpecifications#1]
reported by Anu.

The current attribute 'structure' is probably too narrow and its definition seems to imply either Protein or Nucleic Acid

**Comment by agbeltran** [comment]
@proccaserra what term do you suggest to change 'structure' to?

**Comment by proccaserra** [comment]
@agbeltran I presume we need to add 'hasCharacteristics' attribute from 'Material Entity' to the table describing the 'Molecular Entity' object. I believe we assumed 'molecular entity' would be a specialized class of 'Material Entity' and the 'structure' attribute be a special type of 'characteristic'. 

since  allOf can not be used to “extend” a schema to add more details to it in the sense of object-oriented inheritance (taken from http://spacetelescope.github.io/understanding-json-schema/reference/combining.html)

in other words, we need to be able to specify 'chemical structure ' information about a molecule if needed but also be able to provide any attribute about that molecule if need be.

**Comment by agbeltran** [comment]
Indeed, JSON schemas don't have inheritance in the OO sense, but we are still using 'allOf' to support extensions.

For this, it seems that the simplest way would be to extend the range of 'hasCharacteristics' property from 'Dimension' to 'Dimension' or 'Material' (and also use it in 'Molecular Entity'). 

Thanks!

**Comment by agbeltran** [comment]
This has been implemented.
